lua/nvim-treesitter/query.lua

```diff
@@ -69,6 +69,28 @@ function M.get_matches(bufnr, query_group)
   return query_cache[query_group][bufnr].cache
 end
 
+local function filter_files(file_list)
+  local main = {}
+  local after = {}
+
+  for _, fname in ipairs(file_list) do
+    if string.find(fname, "after", 1, true) then
```

Comment:
Is the occurence of "after" really the only possibility to detect after directories?

Reply:
Hehe that was a bit hacky I admit

Reply:
It should be better now, still needs documenting.